### PR TITLE
Added registration info

### PR DESCRIPTION
**Pull request by jdamerow**

### CODE REVIEW

File: pages/registration.md [hunk 1]
```diff
@@ -4,10 +4,8 @@ title: Registration
 permalink: /registration/
 ---
 
-## Registration is not open yet
+## Registration is now open!
 
-Registration hasn't open yet. Please check back later.
+As our previous meetings and community calls, the virtual workshop will be held on Zoom. You can register for free [here](https://forms.gle/zEPeEK5Lo33BcpbD7). You will receive a copy of your registration to the email address you provide. Please check your spam folder if you don't receive one.
```
Comment (cosden) [inline pages/registration.md]:
```suggestion
As our previous meetings and community calls, the virtual workshop will be held on Zoom. You can register for free [here](https://forms.gle/zEPeEK5Lo33BcpbD7). The zoom meeting registration link is on the second page - don't forget to do this too. You will receive a copy of your registration to the email address you provide. Please check your spam folder if you don't receive one.
```

Comment (cosden) [reply pages/registration.md]: I tried out the form and was a bit concerned that I wasn't getting the zoom registration link. Not sure this is the best way to say it, but I think some kind of reminder or warning that they need to do both (so people don't just close the window after clicking submit on the form) would be helpful.

**Review by cosden — APPROVED** [review]
I made a suggestion, but would be happy to merge this and get it live asap (i.e. before the community call) with or without the addition.

**Comment by jdamerow** [comment]
How is that?

**Comment by cosden** [comment]
Looks great!

**Review by cosden — APPROVED** [review]

**Pull request merged**